{"name": "test_log_in_wrong_password", "status": "broken", "statusDetails": {"message": "_pytest.fixtures.FixtureLookupError: ('setup', <FixtureRequest for <Function test_log_in_wrong_password>>)", "trace": "file /home/alina/PycharmProjects/gmail3/test_project/tests/log_in_test.py, line 15\n      def test_log_in_wrong_password(self):\nE       fixture 'setup' not found\n>       available fixtures: cache, capfd, capfdbinary, caplog, capsys, capsysbinary, doctest_namespace, monkeypatch, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory\n>       use 'pytest --fixtures [testpath]' for help on them.\n\n/home/alina/PycharmProjects/gmail3/test_project/tests/log_in_test.py:15"}, "start": 1567941475510, "stop": 1567941475510, "uuid": "297d2598-efa2-4272-9038-8844a1372698", "historyId": "800788d7ea7d2bb0ef4c9f2e88ef9aee", "fullName": "test_project.tests.log_in_test.TestLogIn#test_log_in_wrong_password", "labels": [{"name": "parentSuite", "value": "test_project.tests"}, {"name": "suite", "value": "log_in_test"}, {"name": "subSuite", "value": "TestLogIn"}, {"name": "host", "value": "alina-Inspiron-N5050"}, {"name": "thread", "value": "25758-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_project.tests.log_in_test"}]}